e2e_tests/e2e_test_setup.sh

```diff
@@ -1,8 +1,15 @@
 set -e
 
-# This script is for CI purposes only. If you want to run the e2e tests locally make sure to setup your environment accordingly using:
-#          curl https://travis.nigiri.network | bash
-#          docker-compose -f ./docker-compose.yml up -d
+# This script is primarily for CI purposes. If you want to run the e2e tests locally use the -L flag:
+#    ./e22_test_setup.sh -L
+
+if getopts ":L" arg; then
+  mkdir nigiri
```

Comment:
This will fail if it already exists right?

Reply:
You're correct, oops! Lemme adjust that.